src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -257,6 +261,7 @@ public Command followTrajectory(String name, PathPlannerTrajectory trajectory){
     @Override
     public void periodic() {
         odometer.update(getRotation2d(), getModulePositions());
+        SmartDashboard.putNumber("GyroPitch", getPitch());
```

Comment:
Add this to the shuffleboard tab instead of using SmartDashboard.

Reply:
Done